Homework/Develop/index.html

```diff
@@ -22,61 +23,69 @@ <h1>Hori<span class="seo">seo</span>n</h1>
                     <a href="#social-media-marketing">Social Media Marketing</a>
                 </li>
             </ul>
-        </div>
+
+        </nav>
+    </header>
+
+    <div class="hero">
+
     </div>
-    <div class="hero"></div>
-    <div class="content">
-        <div class="search-engine-optimization">
-            <img src="./assets/images/search-engine-optimization.jpg" class="float-left" />
+    <Main>
```

Comment:
Are you sure it is `Main`. Remember that all HTML tags are lowercase. Please correct.

Reply:
thanks, made the changes in here 8b32038a16205c6727065cc1712ce5cd6c7a0285